### ISSUE TITLE
rel type should be int rather than uint

### ISSUE THREAD

**Issue by thomas-fossati** [sacmwg/draft-ietf-sacm-coswid#31]
In the CDDL, `rel` is defined as:
```
$rel /= uint / text
```
but the textual description that follows allows negative values (for testing and use in closed environments).

So I guess it should instead say:
```
$rel /= int / text
```
or even
```
$rel /= -256..65535 / text
```

**Comment by henkbirkholz** [comment]
Thank you!

'$rel /= -256..65535 / text'

sounds fine. Will be included in the next proposal.

**Comment by david-waltermire** [comment]
I fixed this in PR #35.
